src/components/footer.jsx

```diff
@@ -3,31 +3,44 @@ import React from 'react';
 class Footer extends React.PureComponent {
   render() {
     return (
-      <footer className="usa-footer usa-footer-slim" role="contentinfo">
-        <div className="usa-footer-primary-section">
-          <div className="usa-grid-full">
-            <nav className="usa-footer-nav usa-width-two-thirds">
-              <ul className="usa-unstyled-list">
-                <li className="usa-width-one-fourth usa-footer-primary-content">
-                  <a className="usa-footer-primary-link" href="https://qpp.cms.gov/resources/developers">QPP Developer Resources</a>
-                </li>
-              </ul>
-            </nav>
-            <div className="usa-width-one-third">
-              <div className="usa-footer-primary-content usa-footer-contact_info">
-                <p>(800) CALL-GOVT</p>
+      <footer>
+        <div className="container">
+          <div className="row">
+            <div className="col-sm-8 col-md-9 col-lg-9">
+              <div className="logo-container">
+                <img className="qpp-logo" src="https://qpp.cms.gov/images/qpp_logo_reversed.png" alt="qpp logo"></img>
               </div>
-              <div className="usa-footer-primary-content usa-footer-contact_info">
-                <a href="mailto:info@agency.gov">info@agency.gov</a>
+              <div className="links">
+                <ul>
+                  <li>
+                    <a href="https://qpp.cms.gov/developers" data-track-category="FooterNav" data-track-action="OpenDeveloperTools" data-track-label="Quality Payment Program">Developer Tools</a>
```

Comment:
Good question! As far as I can tell... nothing. I was just copying them from the qpp-style header/footer, but that's a bad reason to use them. Removed them in 07ff302961da611d50249f4c276b9c3ce99c2281!